concourse/pipelines/deploy-cloudfoundry.yml

```diff
@@ -464,6 +507,117 @@ jobs:
         params:
           file: cf-deploy/cf-manifest.yml
 
+  - name: post-deploy
+    plan:
+    - aggregate:
+      - get: pipeline-trigger
+        passed: [ 'deploy']
+        trigger: true
+      - get: paas-cf
+        passed: ['deploy']
+      - get: cf-secrets
+        passed: ['generate-manifest']
+      - get: cf-manifest
+        passed: ['deploy']
+      - get: graphite-nozzle
+    - task: retrieve-config
+      config:
+        platform: linux
+        inputs:
+        - name: paas-cf
+        - name: cf-secrets
+        - name: cf-manifest
+        outputs:
+        - name: config
+        params:
+        image: docker:///ruby#2.2-slim
+        run:
+          path: sh
+          args:
+          - -e
+          # FIXME
+          - -x
+          - -c
+          - |
+            VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
+
+            CF_ADMIN=admin
+            CF_PASS=$($VAL_FROM_YAML secrets.uaa_admin_password cf-secrets/cf-secrets.yml)
+            FIREHOSE_USER=graphite-nozzle
+            FIREHOSE_PASS=$($VAL_FROM_YAML secrets.uaa_clients_firehose_password cf-secrets/cf-secrets.yml)
+            API_ENDPOINT=$($VAL_FROM_YAML properties.cc.srv_api_uri cf-manifest/cf-manifest.yml)
+            UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+            DOPPLER_ENDPOINT=$($VAL_FROM_YAML properties.loggregator.traffic_controller_url cf-manifest/cf-manifest.yml)
+            GRAPHITE_SERVER=$($VAL_FROM_YAML jobs.graphite_z1.networks.cf1.static_ips.0 cf-manifest/cf-manifest.yml)
+
+            for var_name in CF_ADMIN CF_PASS FIREHOSE_USER FIREHOSE_PASS API_ENDPOINT UAA_ENDPOINT DOPPLER_ENDPOINT GRAPHITE_SERVER; do
+              echo export "${var_name}"=\"$(eval echo \$${var_name})\"
+            done > config/config.sh
+
+            # FIXME
+            cat config/config.sh
+
+    - task: deploy-graphite-nozzle
```

Comment:
Why don't we use CF resource to deploy an app ? https://github.com/concourse/cf-resource